src/way/way.pug

```diff
@@ -10,7 +10,7 @@ nav.way
       a.way_link( href="https://github.com/postcss/postcss#articles" )
         | Learn
     li.way_item
-      a.way_link( href="h/docs/postcss-plugins" )
+      a.way_link( href="docs/postcss-plugins" )
```

Comment:
```suggestion
      a.way_link( href="/docs/postcss-plugins" )
```

Reply:
Whoops, got it, should be corrected now.